src/braket/schema_common/schema_base.py

```diff
@@ -75,6 +76,10 @@ def parse_raw_schema(json_str: str) -> BraketSchemaBase:
         schema = BraketSchemaBase.parse_raw(json_str)
         module = BraketSchemaBase.import_schema_module(schema)
         name = schema.braketSchemaHeader.name
-        class_name = "".join([s.capitalize() for s in name.split(".")[-1].split("_")])
+
+        def capitalize_first_alpha(string):
```

Comment:
[No action needed] We should also consider performance when evaluating alternative solutions. Sticking with existing method and naming the schema classes as 2000q would have led to a simpler implementation. This new implementation takes ~4.15x time than the old implementation (predominantly due to the use of regex). This shouldn't be a big issue in this specific case, but might be worth keeping in mind in the future as these things add up rather quickly.